### ISSUE TITLE
Link preview don't work[BUG]

### ISSUE THREAD

**Issue by IanosJunior** [adiwajshing/Baileys#1510]
The message was sent but did not arrive with the image.

{
    "key": {
        "remoteJid": "553799552539@s.whatsapp.net",
        "fromMe": true,
        "id": "BAE5C4FEE85D068B"
    },
    "message": {
        "extendedTextMessage": {
            "text": "Hi, this was sent using https://youtu.be/Isj-SGbves8",
            "matchedText": "https://www.youtube.com/watch?v=Isj-SGbves8&feature=youtu.be",
            "canonicalUrl": "https://www.youtube.com/watch?v=Isj-SGbves8&feature=youtu.be",
            "description": "Vídeo de apresentação do sistema Via Rápida Comunica para WhatsApp.Nele são apresentadas algumas características do sistema, como:- Vários computadores conve...",
            "title": "Apresentação Via Rápida Comunica para WhatsApp",
            "previewType": "NONE",
            "jpegThumbnail": "/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wAARCACQAMADASIAAhEBAxEB/8QAGwABAAEFAQAAAAAAAAAAAAAAAAcBAgMFBgT/xABCEAACAQICBgUHCQgCAwAAAAAAAQIDEQQTEhchMVGTBVJTVFUiMkFhkZLRBgcUFjRxcoGhFSMkMzVzscFigmOi4f/EABoBAQADAQEBAAAAAAAAAAAAAAACAwQFAQb/xAAgEQEAAwEBAQACAwEAAAAAAAAAAQIDEQQSEzEUIVFh/9oADAMBAAIRAxEAPwCPwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAdkvm9xbS/i4L1Zb+JXV5i7/bafLZ3lahTdWcpKq3fa4xXpLIYWhUnouFVX9MoKweOH1d4rv1PlsausV36ny2SDTwcaTbpzlFvY7JGXKn28/0POnUc6usV36ny2NXWK79T5bJGypdtP8AQZUu2n+g6I51dYrv1PlsausV36ny2SPly7Wf6DLl2s/0HRHGrrFd+p8tjV1iu/U+WyR8uXay/QZcu1l+h69Rxq6xXfqfLY1dYrv1PlskjQl2khoS7SQEb6usV36ny2NXWK79T5bJI0JdpIaEuuwI31dYrv1PlsausV36ny2STovrsaL67AjbV1iu/U+Wxq6xXfqfLZJOi+sytn1mBGurrFd+p8tjV1iu/U+WySrPrCz4gRrq6xffqfLY1dYvv1Plskqz4lJNxV9r9SQEbau8V36ly2Wv5vsVe302ny38SRasoTeXKnJ3V72/2UpRotxtSavtTa3AeiKVtxWy4IpHzS4CllwQsuCKlAFlwQsuCFxcBZcELLgg2ktuwALLghZcEAAsuCFlwQACy4Ip5PqDew4nEVpzqy2tNN7bvaX44zrMx1m9HojCInneu28n1Cy9RwmnPry9p1nQcpS6LpOTbe1Xf3ktvPOUd6h5/XG1vnnGwsuCFlwQKmZsUsuCFlwRUAUsuCFlwRUAUsuCLZpW3ekvLZ7vzAR80uLY+aXADHW/lSMhjrfy5CHk/pTIp8H7zGRT4P3mZCo6chhlh6cotNOzVvOZXIp8H7zMoHTkMWRT4P3mMinwfvMyPcc7jOm8TSxFSFJwtGTjZx3WLM87aTyqrXWmUds32RT4P3mMinwfvM5l9P462zK903fRGLqY3B5tWykpNbPST0wvnHbK8vTnrPzV6amEpVI2km0mn5zOLlJwqza4s7o8EuhsDKTk6O17fOZLz7Rn36Q9Xmttz5cnKpKSs3s+46roH+lUvvf+Sv7FwHY/+zPZQoU8PSjTpRUYLcie+9NK/NYQ8vlvlebWllABjdAAAAAAC2e78y4sqOyXraQFY+aXFsfNLgKGOu0qcrsyGm6Vf8Vb/iinbX8Vfrieef5J+WzlOTqOMZRSST2q/wDsqsx7pwf/AFfxOePf0LHRrVrPermbH2/kvFfni7TzfFfrrZKNa7vKFvR5L+JW1XrQ91/Eyg3svHnqRxDptU501L0Nxdv8nIYz7dW09v7x3ts9J2xosT8n51sTUqRrxSnJys4vYa/LpWkz9MPtxvpWPiOtDJwt5Kkn62dL8nf6c/7jPH9W6veIe6zbdGYJ4HC5TmpvScm0rFvo2panKyp8mGlNPq0ce0AHPdUAAAoVKMCjklvaGnHrL2mk6Rk3i6i9aPHKOlba1Y59/d83mvz+muvm+qxPXUJp7ncqavoRWhVV29qZtDZlp+SkWZ71+LfIY6u6P4kZDHV3R/EixBdHzS4tj5pcBQ0vSv2z/qjdGKrhqNaWlUpqT3XZR6Mp1p8wtyvFLdlz+k7WvsPb0S2q1RqOl5PE9qweHzWsqNrL/ZnpYelRvlwUb77GXHx3z0i0z+l2norek1iDMn2T9qGZPsn7UZAdFj4x5k+yftQzJ9k/ajI9xocX07Vo15whTpyUZOO299hZTO2k8rCrXWuUdtLdZk+yftRSVWoraNFvbt2o0D+UeItsoUr/AJm36LxssdhM2cVCWk1ZEr43zjtoQz9Ges8rL05k+yftQzJ9k/ajIYMbX+jYWpWST0VfbuKo/ueL5nkdlfmT7J+1DMn2T9qNBL5Q11JpUqTXHaW/WPEJq9Cnb02uaP42vO8Zf5uP+uhzJ9k/aik6lRQbVFtpbFpLaZVtSfEWMzXz/rn8ZKUsTJzg4Sdrxve2wwybkkm3ZHQVMLQqz0p04uXEt+g4bsYnM08V7Xm0T+22vorFYiYeTobdV+9G0MdKjTopqnBRT32MhuxznOkVlm0t92m0Bjq7o/iRkLJ7l95agrHzS4tj5pcAAAGJxmqjlFRd0ltYk62i7Rjf8X/wylA84xp1rbYwv97+BW9Xqw95/AvKg4wVXiMt5cael6Lydv8AByGN0PptfT0lLTd7bt52r3HPYroHE1cVVqQqU9GcnJXbvtNflvWsz9Txh9uV71j5jrS/uuM/0Ok+T/2CWgvJzHve01/1dxfXpe1/A3PRWCngsJlVJRcnJy8ncW+nSlqcrKjx46U07avHs8vhH2ng6YzP2dX2R0NDbt270bI8+Pw8sVg6tGLSlONk2YKf1aJdPSO0mI/xx0VDRWk4N+tstqKKXkuP5Nmz+ruL7Sl7X8Cq+TuKvtqUl+b+B1/zZ/v6cH+Nt+vl0sPMj9xcWxVklwLjjvoQAAAAALZ7vzLi2e78wLVNJWK5kSONY2J8Po8yQ1jYnw+jzJASPmRGZEjjWNifD6PMkNY2J8Po8xgSPmRGZEjjWNifD6PMkNY2J8Po8yQEj5kSjqpLiRzrGxPh9HmSGsbE+H0eYwJFzl1WM5dVkdaxsT4fR5jGsbE+H0eYwJFzl1WM5dVkdaxsT4fR5jGsbE+H0eYwJFzl1WM5dVkdaxsT4fR5jGsbE+H0eYwJFzl1WM5dVkdaxsT4fR5jGsbE+H0eYwJHzIjMiRxrGxPh9HmMaxsT4fR5kgJHzIjMiRxrGxPh9HmMaxsT4fR5kgJHzIjMiRxrGxPh9HmSKP5xcS1/T6PMYEkZkS2U1LYiNl84OITb/Z9Lb/5ZCPzg14y0l0fSv/dkBxoAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAP/2Q=="
        }
    },
    "messageTimestamp": "1650398308",
    "status": "PENDING"
}

**Comment by Fabrizz** [comment]
The workaround for this is to forward a message that contains the data.
```
                {
                    forward: {
                        key: { fromMe: true },
                        message: {
                            extendedTextMessage: {
                                title: "link title",
                                text: ">> https://link.io/",
                                matchedText: "https://link.io/",
                                canonicalUrl: "https://link.io/",
                                description: "link description",
                                jpegThumbnail: fs.readFileSync("../tests/media/thumb.jpg")
                            }
                        }
                    }
                }
```

**Comment by IanosJunior** [comment]
> The workaround for this is to forward a message that contains the data.
> 
> ```
>                 {
>                     forward: {
>                         key: { fromMe: true },
>                         message: {
>                             extendedTextMessage: {
>                                 title: "link title",
>                                 text: ">> https://link.io/",
>                                 matchedText: "https://link.io/",
>                                 canonicalUrl: "https://link.io/",
>                                 description: "link description",
>                                 jpegThumbnail: fs.readFileSync("../tests/media/thumb.jpg")
>                             }
>                         }
>                     }
>                 }
> ```
> 
> EDIT: this is the fix [872536f](https://github.com/adiwajshing/Baileys/commit/872536fa5bad590846866ddf8dd2855b66360ee8)

I used the fix but don't work.

How to send this message?

```
{
                    forward: {
                        key: { fromMe: true },
                        message: {
                            extendedTextMessage: {
                                title: "link title",
                                text: ">> https://link.io/",
                                matchedText: "https://link.io/",
                                canonicalUrl: "https://link.io/",
                                description: "link description",
                                jpegThumbnail: fs.readFileSync("../tests/media/thumb.jpg")
                            }
                        }
                    }
                }
```

Do you have a example?

**Comment by MaznAbdullah** [comment]
any update ?
